modules/special-rules/pages/shooting-at-war-engines.adoc

```diff
@@ -9,8 +9,8 @@ These categories reflect the fact that these units are so large and easy to hit.
 
 * A War Engine unit can become Immobilised -- and so even easier to hit -- as a result of *critical damage*.
 * There is no target category for 'Immobilised War Engine in the open' -- instead, you simply roll as many hit dice as the total Firepower that you direct at the unit.
-
-{blank}[TODO: Cross-reference the Firepower table, once we've added it. Also, we actually target detachments not units, and we need to clarify what happens when a War Engine detachment has a mix of Immobilised and non-Immobilised WE units.]
+* If a target detachment contains both mobile and Immobilised War Engine units, then you may choose to shoot at only those that are Immobilised, if you wish.
+If you do this then you may allocate hits to the Immobilised units **only**.
```

Comment:
Damn - in previous suggestion I accidentally inserted the GitHub double-asterisk markup for bold, rather than the Asciidoc single asterisk markup.

```suggestion
If you do this then you may allocate hits to the Immobilised units *only*.
```